backend.js

```diff
@@ -145,51 +144,50 @@ app.post('/api/login', async (req, res) => {
  *  a. Foreign key user_id_fk matches those found in users, authentication tables
  *  b. Default values: Theme = "Light", View = "List"
  * 
- * ToDo: 
- * - Implement security features on username and password limits to test
+ * ToDo: Implement Email into account creation
  *
- * Returns JSON object with boolean value for account creation
+ * Returns JSON object with String status for account creation
  */
 app.post('/api/create_account', async (req, res) => {
   username = req.body.username
   password = req.body.password
+  email = req.body.email
   startsalt = "food"
   endsalt = "star"
-  status = {status : false}
-
-  users = await Authentication.findAll({ where: { "username": username } });
-  if (users[0] != null) {
-    console.log("Username " + username + " already in use.")
-    res.send(status)
-  }
-  else {
-    createUser = await Users.create(req.body)
-    user_id_fk = createUser.user_id
-    //console.log("username: " + username + "\nPassword: " + password)
-
-    // Hash / salt password and create Authentication entry
-    hashfunc = crypto.createHash('ripemd160')
-    hashfunc.update(startsalt.concat(password, endsalt))
-    password = hashfunc.digest('base64');
-    //console.log("Hash = " + password)
-    createAuth = await Authentication.create({username, password, startsalt, endsalt, user_id_fk})
-
-    //Create Settings Entry
-    theme = "Light"
-    view = "List"
-    createSettings = await Settings.create({theme, view, user_id_fk})
-    status.status = true
-    res.send(status)
+  status = {status : "Error Creating Account."}
```

Comment:
I think we should also have a boolean success field in the status, because otherwise the frontend will have to check the text strings which creates room for error